package.json

```diff
@@ -7,10 +7,15 @@
     "start": "nodemon src/index.js",
     "test": "ava",
     "deploy-api": "firebase use api; firebase deploy --only functions",
-    "deploy-rules": "firebase use production; firebase deploy --only rules"
+    "deploy-rules": "firebase use staging; firebase deploy --only database",
+    "deploy-rules-production": "firebase use production; firebase deploy --only rules"
   },
   "devDependencies": {
     "ava": "^0.21.0",
     "nodemon": "^1.11.0"
+  },
+  "dependencies": {
+    "cors": "^2.8.4",
+    "firebase": "^4.3.0"
```

Comment:
since these deps are for our app they should be moved to src/package.json. Also, is `firebase` needed? there is already `firebase-admin`.